pandas/io/formats/format.py

```diff
@@ -53,35 +53,38 @@
     Parameters
```

Comment:
Parameters should go before the See Also section, right after the Extended Summary.